src/openvic-simulation/types/Sliders.hpp

```diff
@@ -0,0 +1,44 @@
+#pragma once
+
+#include "openvic-simulation/types/fixed_point/FixedPoint.hpp"
+#include "openvic-simulation/utility/Getters.hpp"
+
+namespace OpenVic {
+  // TODO: make struct for tariff slider, unlike other sliders, you can have negative values
+  struct slider_value_t {
+  friend struct  SliderManager;
+  private:
+      int PROPERTY(min);
+      int PROPERTY(max);
```

Comment:
Is it enough to have just `value`, `min` and `max`? Don't we need another pair of final min/max values to act we limits on potentially shifting regular min/max values (e.g. so that stacking modifier effects doesn't let you move a slider outside [0,100]), probably defined as `const int_type PROPERTY(final min/max variable name)` OR they could be define via template parameters similar to what you had before (it will work with primitive types like `int32_t`)

Reply:
This was the original plan, but the tariff slider makes one edge case because unlike every other slider, it has the bounds [-100,100] and I couldn't figure out the syntax in `CountryInstance` to make it so I can specify a different "`strict_min`". I can implement this and consult on the server for the syntax